Airports/EGNS/Positions.txt

```diff
@@ -1,5 +1,5 @@
 EGNS_A_APP:Ronaldsway Approach:135.905:NSA:S:EGNS:APP:-:-:4550:4567:N054.05.00.000:W004.37.26.000
-EGNS_APP:Ronaldsway Radar:120.855:NSR:S:EGNS:APP:-:-:4550:4567:N054.05.00.000:W004.37.26.000
-EGNS_F_APP:Ronaldsway Director:125.305:NSF:S:EGNS:APP:-:-:4550:4567:N054.05.00.000:W004.37.26.000
+EGNS_APP:Ronaldsway Radar:135.905:NSR:S:EGNS:APP:-:-:4550:4567:N054.05.00.000:W004.37.26.000
+EGNS_F_APP:Ronaldsway Director:120.855:NSF:S:EGNS:APP:-:-:4550:4567:N054.05.00.000:W004.37.26.000
```

Comment:
Is the callsign for EGNS_F_APP one correct? [Here](https://drive.google.com/file/d/16Pe_ksn-vnuF-mnMNmBWENuFG1LDcErl/view) it's refered to as "Ronaldsway Radar".